docs/sphinx/using/cudaq/platform.rst

```diff
@@ -72,14 +72,14 @@ Here is an example.
     :end-before: [End Documentation]
 
 
-One can then target the :code:`nvidia-mqpu` platform by:
+One can then target the :code:`nvidia-mqpu` platform by executing the following commands:
 
 .. code-block:: console 
 
     nvq++ observe_mqpu.cpp -target nvidia-mqpu
     ./a.out
 
-Equivalently, in Python
+Equivalently, in Python, we would use the following:
```

Comment:
"we"

Are you removing first-person?

Reply:
I was attempting to complete the sentence.  I didn't focus on 1st person versus "one" in the document.  But this does bring up a good point that we should be consistent.  According to the style guide, the writing should be conversational, which I think lends itself to 1st person.  I will confirm with the style team.